src/cc_catalog_airflow/dags/common/urls.py

```diff
@@ -64,7 +64,8 @@ def rewrite_redirected_url(url_string):
         response = requests.get(url_string)
         if response.ok:
             rewritten_url = response.url
-            logger.info(f'{url_string} was rewritten to {rewritten_url}')
+            if rewritten_url != response.url:
```

Comment:
Perhaps the intention was to compare with the original `url_string`?
```suggestion
            if rewritten_url != url_string:
```

Reply:
Thank you for catching this, @krysal !